content/zh/releases/patch-releases.md

```diff
@@ -0,0 +1,312 @@
+---
+title: 补丁发布
+type: docs
+---
+<!-- 
+title: Patch Releases
+type: docs
+-->
+
+<!-- 
+Schedule and team contact information for Kubernetes patch releases.
+
+For general information about Kubernetes release cycle, see the
+[release process description].
+-->
+Kubernetes 补丁发布的时间表和团队联系信息。
+
+有关 Kubernetes 发布周期的常规信息，请参阅[发布过程说明]。
+
+<!-- 
+## Cadence
+
+Our typical patch release cadence is monthly. It is
+commonly a bit faster (1 to 2 weeks) for the earliest patch releases
+after a 1.X minor release. Critical bug fixes may cause a more
+immediate release outside of the normal cadence. We also aim to not make
+releases during major holiday periods.
+-->
+## 节奏
```

Comment:
```suggestion
## 节奏   {#cadence}
```

Reply:
do we always need an anchor for title in zh ?

Reply:
It is better to add anchor because we don't know which subtitle is referenced somewhere else.